src/components/sections/pricing/plans/components/login-info/index.tsx

```diff
@@ -0,0 +1,35 @@
+'use client';
+
+import { logOut } from '@httptoolkit/accounts';
+
+import { StyledLoginInfoWrapper } from './login-info.styles';
+import type { LoginInfoProps } from './login-info.types';
+
+import { Button } from '@/components/elements/button';
+import { Link } from '@/components/elements/link';
+import { Text } from '@/components/elements/text';
+
+
+
+
+export const LoginInfo = ({ isLoggedIn, email }: LoginInfoProps) => {
+  if (!isLoggedIn) {
+    return (
+      <Text fontSize="m" textAlign="center" color="darkGrey">
+        Want to manage an existing account? Log into your dashboard at 
+        <Link href="mailto:accounts.httptoolkit.tech">accounts.httptoolkit.tech</Link>.
```

Comment:
This is a link not an email
```suggestion
        <Link href="https://accounts.httptoolkit.tech">accounts.httptoolkit.tech</Link>.
```